kernel-headers/rdma/irdma-abi.h

```diff
@@ -1,6 +1,6 @@
-/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR Linux-OpenIB */
+/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR Linux-OpenIB) */
 /*
- * Copyright (c) 2006 - 2021 Intel Corporation.  All rights reserved.
+ * Copyright (c) 2006 - 2022 Intel Corporation.  All rights reserved.
```

Comment:
If you decided to change it, we are in 2023 now.

Reply:
Please use kernel-header/update script to generate this commit, and this patch should be first in the series.

Reply:
Thanks for the feedback.